### ISSUE TITLE
libXfixes missing dependencies

### ISSUE THREAD

**Issue by nathanhaigh** [hpcugent/easybuild-easyconfigs#3166]
Just tried installing libXfixes 5.0.1 for foss-2015b but had some weird config/build errors. I was able to fix it and successfully install it by adding the following additional dependencies:

```
    ('libX11', '1.6.3', '-Python-2.7.10'),
    ('kbproto', '1.0.7'),
    ('xproto', '7.0.28'),
```

**Comment by boegel** [comment]
@nathanhaigh seems like more dependencies are missing (`libXau`, `libxcb`) to avoid picking stuff up from the OS:

```
$ ldd $EBROOTLIBXFIXES/libXfixes/5.0.1-intel-2016a/lib/libXfixes.so | grep -i libx
	libX11.so.6 => /usr/lib64/libX11.so.6 (0x00002b61f2c5d000)
	libxcb.so.1 => /usr/lib64/libxcb.so.1 (0x00002b61f555f000)
	libXau.so.6 => /usr/lib64/libXau.so.6 (0x00002b61f577e000)
```

BTW: `kbproto` and `xproto` are only build dependencies.

**Comment by verdurin** [comment]
This is what's hitting me while trying to build `RELION` #2653

**Comment by verdurin** [comment]
@boegel - `libXau` and `libxcb` would be provided by `libX11`, wouldn't they?

**Comment by wpoely86** [comment]
@verdurin take the one from #3222, it works and has all the correct deps.

**Comment by verdurin** [comment]
@wpoely86 - yes, it does work - thanks.

**Comment by boegel** [comment]
fixed with #3222 for `intel/2016a` and in #3308 for `foss/2016a`, so closing this
